From afbca92c40b839855c92120c6963ad0ec2f36047 Mon Sep 17 00:00:00 2001
From: Josh Wilsdon <josh@wilsdon.ca>
Date: Fri, 12 Oct 2018 13:36:48 -0700
Subject: [PATCH] OS-7306 /usr/vm tests should include test name in output
 Reviewed by: Dave Eddy <dave.eddy@joyent.com> Approved by: Robert Mustacchi
 <rm@joyent.com>

---
 src/vm/runtest | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/vm/runtest b/src/vm/runtest
index b87ce942..086f568c 100755
--- a/src/vm/runtest
+++ b/src/vm/runtest
@@ -38,6 +38,8 @@ if [[ -z "$1" ]]; then
     exit 1
 fi
 
+echo "# Running $1"
+
 # Before starting tests, ensure we have the source(s) we want for test VM images
 
 have_sources="imgadm sources | sort"
-- 
2.21.0

